src/frontend/komponenter/Fagsak/Saksoversikt/PersonUtbetaling.tsx

```diff
@@ -1,34 +1,47 @@
 import React from 'react';
 
-import { BodyShort } from '@navikt/ds-react';
+import styled from 'styled-components';
+
+import { BodyShort, HStack } from '@navikt/ds-react';
+import { ASpacing2, ASpacing4, ASpacing8 } from '@navikt/ds-tokens/dist/tokens';
 
 import type { IUtbetalingsperiodeDetalj } from '../../../typer/vedtaksperiode';
 import { formaterBeløp } from '../../../utils/formatter';
-import DashedHr from '../../Felleskomponenter/DashedHr/DashedHr';
 import PersonInformasjon from '../../Felleskomponenter/PersonInformasjon/PersonInformasjon';
 
+const Ytelser = styled.section`
+    margin: ${ASpacing2} 0 ${ASpacing4} ${ASpacing8};
+    border-bottom: 1px dashed;
+`;
+
+const Ytelselinje = styled(HStack)`
+    margin-bottom: ${ASpacing4};
+`;
+
 interface IPersonUtbetalingProps {
     utbetalingsperiodeDetaljer: IUtbetalingsperiodeDetalj[];
 }
 
 const PersonUtbetaling: React.FC<IPersonUtbetalingProps> = ({ utbetalingsperiodeDetaljer }) => {
     return (
-        <li>
+        <section>
```

Comment:
Hvorfor passer det bedre med nøstede sections her fremfor å bruke div? 😊 prøvde å se hvordan det ser ut, men dolly/pdl er nede akkurat nå ser det ut som, så får ikke sjekket 😞

Reply:
Visuelt ser det likt ut, men sånn jeg har skjønt det så sørger `section` bla. for bedre navigering (feks med skjermleser). @kristeine er mer stødig her enn meg, jeg har gjort det likt her som vi gjorde i ba-sak hehe

Reply:
Hmm, jeg tror jeg er litt uenig på denne. Semantisk synes jeg det hadde gitt mer mening dersom section hadde vært hele "løpende utbetaling"s-bolken, og at seksjonen var "aria-labelledBy" headeren. Seksjoner skal bare unntaksvis ikke ha header. Så tenker jeg det som er listet opp i selve seksjonen fortsatt kunne vært en liste. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Jeg trooor også at section minimum trenger aria-label/aria-labelledBy for at den skal virke som ønsket for alle skjermlesere (leses opp/navigeres som en region i dokumentet). Fant denne oversikten: https://www.accessibilityoz.com/2020/02/html5-sectioning-elements-and-screen-readers/ og testet det med VoiceOver, og da fikk jeg den kun opp som en region dersom jeg la på en aria-label.

Jeg godkjenner PR'en men dere må gjerne ta en diskusjon på det eventuelt 😊

Reply:
Enig i det! Daphne og jeg konkluderte på slack om at vi merger PR sånn den er nå og tar en diskusjon på kontoret en dag 😊